udata/core/dataset/models.py

```diff
@@ -360,8 +360,8 @@ class Dataset(WithMetrics, BadgeMixin, db.Owned, db.Document):
                                   default=datetime.now, required=True)
     title = db.StringField(required=True)
     acronym = db.StringField(max_length=128)
-    slug = db.SlugField(max_length=255, required=True,
-                        populate_from='title', update=True)
+    slug = db.SlugField(max_length=255, required=True, populate_from='title',
+                        update=True, follow=True)
```

Comment:
What does `follow` do?

Reply:
It's a custom trigger for the whole `SlugFollow` logic, right?

Reply:
Yes `follow=True` enable redirections following slug changes. I kept the parameter name as it was